vivarium_readdy/util.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+
+
+monomer_ports_schema = {
```

Comment:
I would name this file `schema.py` instead of `utils`. It seems like a file that could be expanded upon with more schemas for reuse.